packages/types/src/dates.ts

```diff
@@ -16,3 +16,11 @@ export type TimeBetweenIntervals = 'milliseconds' | 'seconds' | 'minutes' | 'hou
  * (minutes relative to UTC)
 */
 export type DateTuple = readonly [epoch_millis: number, timezone: number];
+
+export type DateTypes = Date | string | number | DateTuple;
```

Comment:
Not a big fan of this name. It could also use some jsdoc describing what it is used for.

```suggestion
export type DateInputType = Date | string | number | DateTuple;
```